.codespellignore

```diff
@@ -2,3 +2,4 @@ crate
 copyable
 doubleclick
 technics
+egal
```

Comment:
This does not look like a word that we use regularly. Is this just one occurrence?

To avoid false-negatives I suggest to copy the whole line into `.codespellignorelines` instead, if that is the case.